### PR TITLE
Fix sonarcloud

### PR DESCRIPTION
**Pull request by kadel**
fixes SonarCloud integration

/kind tests

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
```
Comment (valaparthvi) [inline .github/workflows/build.yml:5]: Shouldn't this be main?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: Build
+on:
+  push:
+    branches:
```
Comment (valaparthvi) [inline .github/workflows/build.yml:4]: When would this be triggered?

Comment (kadel) [reply .github/workflows/build.yml:4]: yes, my bad

File: .github/workflows/build.yml (hunk 1 above)
Comment (kadel) [reply .github/workflows/build.yml:5]: yes, my bad

**Review by valaparthvi — APPROVED** [review]

**Comment by kadel** [comment]
no point in testing this
/override v4.9-integration-e2e

**Pull request merged**